### ISSUE TITLE
tf.concat issue

### ISSUE THREAD

**Issue by balajiwix** [argman/EAST#134]
Caused by op 'model_0/feature_fusion/concat_2', defined at:
  File "/home/ubuntu/balaji/OCR Extraction/multigpu_train.py", line 182, in <module>
    tf.app.run()
  File "/home/ubuntu/anaconda3/envs/ocrapplication/lib/python3.5/site-packages/tensorflow/python/platform/app.py", line 126, in run
    _sys.exit(main(argv))
  File "/home/ubuntu/balaji/OCR Extraction/multigpu_train.py", line 112, in main
    total_loss, model_loss = tower_loss(iis, isms, igms, itms, reuse_variables)
  File "/home/ubuntu/balaji/OCR Extraction/multigpu_train.py", line 32, in tower_loss
    f_score, f_geometry = model.model(images, is_training=True)
  File "/home/ubuntu/balaji/OCR Extraction/model.py", line 62, in model
    c1_1 = slim.conv2d(tf.concat([g[i-1], f[i]], axis=-1), num_outputs[i], 1)
  File "/home/ubuntu/anaconda3/envs/ocrapplication/lib/python3.5/site-packages/tensorflow/python/ops/array_ops.py", line 1175, in concat
    return gen_array_ops._concat_v2(values=values, axis=axis, name=name)
  File "/home/ubuntu/anaconda3/envs/ocrapplication/lib/python3.5/site-packages/tensorflow/python/ops/gen_array_ops.py", line 625, in _concat_v2
    "ConcatV2", values=values, axis=axis, name=name)
  File "/home/ubuntu/anaconda3/envs/ocrapplication/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/ubuntu/anaconda3/envs/ocrapplication/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/home/ubuntu/anaconda3/envs/ocrapplication/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access
